srcpkgs/jmol/template

```diff
@@ -0,0 +1,28 @@
+# Template for 'jmol'
+pkgname=jmol
+version=14.29.17
+revision=1
+noarch=yes
+maintainer="Brenton Horne <brentonhorne77@gmail.com>"
+homepage="http://jmol.sourceforge.net/"
+depends="virtual?java-environment"
+license="LGPL-3"
```

Comment:
please use SPDX identifier

Reply:
Not sure about the exact identifier, they're a little vague as to the license on the Jmol page so I just went the mailing list a question about this. So I shall fix this when I have the proper information.

Reply:
the esiest way is to look at the source files  LGPL license has a license which tells you which version it is

3.0-or-later or 3.0-only

Reply:
Found it, it's LGPL-2.1-or-later.